src/auth/screens/login.screen.js

```diff
@@ -183,6 +184,22 @@ class Login extends Component {
     this.handleOpenURL({ url });
   };
 
+  onLoadEnd = e => {
+    const url = e.nativeEvent.url;
+
+    if (url === 'https://github.com/session') {
```

Comment:
I'm just thinking out loud: can made a separate method, because this code line two times is used, then the meaning of this check will be more clear?

Reply:
Great advice, thanks! Refactored here: https://github.com/gitpoint/git-point/pull/323/commits/c2c4d565b591eae4b50b05a3ec2ba3b0041d0213